README.md

```diff
@@ -13,7 +13,15 @@ Install the package:
 npm i webpack-serve-overlay
 ```
 
-Then require the overlay at the top of your `index.jsx` (or equivalent):
+Then add the following entry to the [`entry` object](https://webpack.js.org/configuration/entry-context/#entry) in your webpack config:
+
+```javascript
+overlay: 'webpack-serve-overlay/onlyIfProduction'
```

Comment:
I think this file should be called `onlyIfDevelopment`, since the idea is that it's only included if `NODE_ENV === 'development'`, not 'production'.

To me it currently gives the impression that it'll only do something when in production 😉

Reply:
Ah good point. I think I just went with the name you suggested and didn't even think about that haha

Reply:
Ya, I suggested that name b/c I thought the check was `=== 'production`, but it's the other way around XD